deploy/lint

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash -e
 
-BASE_DIR="$(dirname "$(dirname "$(readlink -e "$0")")")" # /esp
+# http://stackoverflow.com/a/4774063
+pushd `dirname $0` > /dev/null
+SCRIPTPATH=`pwd -P`
+popd > /dev/null
 
-echo "Running lint checks..."
+BASE_DIR="$(dirname "$SCRIPTPATH")" # /esp
+
+echo -n "Running lint checks..."
```

Comment:
`-n` is going to make the first line of  `flake8` output look weird, that's why I didn't do it.